src/core/model_utils.py

```diff
@@ -592,10 +599,60 @@ def pre_save(self, model_instance, *args, **kwargs):
             return data
 
 
+class JanewayBleachFormField(BleachFormField):
+    """
+    An override of BleachFormField
+    to avoid the same unwanted effects that
+    JanewayBleachField avoids.
+    """
+
+    widget = TinyMCE
+
+    def to_python(self, value):
+        if value in self.empty_values:
+            return self.empty_value
+        return clean(value, **self.bleach_options)
+
+
+class MiniHTMLFormField(JanewayBleachFormField):
+    """
+    A form field to hold limited HTML phrasing content,
+    generally for use inline or on one line.
+    It uses a much smaller bleach allowlist
+    and loads by default with a minimal TinyMCE widget.
+    It is the default formfield used by JanewayBleachCharField.
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs['allowed_tags'] = get_allowed_html_tags_minimal()
+        kwargs['allowed_attributes'] = get_allowed_attributes_minimal()
```

Comment:
We should only do this if these `kwargs` are not passed by the caller. Overriding the provided arguments does violate [Liskov's substitution principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle). Beyond software engineering principles, it could be useful to be able to dynamically configure an instance of this  field that supports different attributes/tags

Reply:
So I thought about this and tried to design for greater flexibility, but we are hindered by the `formfield` definition in Django Beach, which itself violates the substitution principle:

```
https://github.com/marksweb/django-bleach/blob/d675d09423ddb440b4c83c8a82bd8b853f4603c7/django_bleach/models.py#L42-L61
```

The received kwargs are updated by defaults in the opposite way recommended by Django's docs:

https://docs.djangoproject.com/en/5.0/howto/custom-model-fields/#specifying-the-form-field-for-a-model-field

I don't think we can get around this because whatever custom version of formfield we write will need to call super, which will include Django Bleach's definition.